cmd.go

```diff
@@ -25,22 +25,30 @@ func infocmd(opt Options) error {
 	}
 
 	// Find regular git tags.
-	tags, err := Tags(user, repo, token)
+	foundTags, err := Tags(user, repo, token)
 	if err != nil {
 		return fmt.Errorf("could not fetch tags, %v", err)
 	}
-	if len(tags) == 0 {
+	if len(foundTags) == 0 {
 		return fmt.Errorf("no tags available for %v/%v", user, repo)
 	}
 
-	fmt.Println("tags:")
-	for _, t := range tags {
+	tags := []Tag{}
```

Comment:
Very cool, I think I'd read this before but didn't get the practical implications until now.   Thanks!!!